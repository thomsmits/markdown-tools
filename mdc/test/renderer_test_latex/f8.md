# Chapter
## Slide Footnote

  * entweder eine Quelle angeben<br>*Anna Klinger[^2] erlÃ¤utert, dass der Himmel blau ist.*
 
[^2]: Anna Klinger (2014). "Warum ist der Himmel blau?". Spektrum der Wissenschaft.

