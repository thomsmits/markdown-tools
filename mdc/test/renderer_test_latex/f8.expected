# Chapter
## Slide Footnote

\begin{ul1}
\item entweder eine Quelle angeben\newline\strongenglish{Anna Klinger\footnote{Anna Klinger (2014). \enquote{Warum ist der Himmel blau?}. Spektrum der Wissenschaft.} erl√§utert, dass der Himmel blau ist.}
\end{ul1}